features/analyse_container_image.feature

```diff
@@ -18,10 +20,3 @@ Feature: Analyse Container Image
     Scenario: Query for currently available Analyser Results
         When I query the Result API for a list of analyser results
         Then the list of results should not be empty
-
```

Comment:
why has this scenario been removed?!

Reply:
Two scenarios namely "Submitting a Container Image for analysis" and  "Get the Analyser Results for my last submitted Container Image" from analyse_container_image.feature are correlated and one main scenario . Only once first scenario is passed , we can query the result API for details of the latest analyzer result. So it hasnt been removed but 2 of them have been merged

Reply:
duh, that was not obvious form the PR description :)